ansible/roles/kubernetes-common/tasks/main.yml

```diff
@@ -2,12 +2,18 @@
 - set_fact:
     kubernetes_node_ip: "{{ hostvars[inventory_hostname]['ansible_'~item]['ipv4']['address'] }}"
   with_items:
-    - enp0s8
+    - "{{ kubernetes_common.primary_interface }}"
+  when: kubernetes_common.primary_interface is defined
 
-- name: drop vagrant node ip flag for the kubelet
+- name: create kubelet config directory
+  file:
+    dest: /etc/systemd/system/kubelet.service.d
+    state: directory
+
+- name: drop extra args kubelet config
```

Comment:
do you mean "drop in"? Might use a different verb like append for clarity.

Reply:
I meant "drop", but see what you mean.